pages/osx/auvaltool.md

```diff
@@ -0,0 +1,12 @@
+# auvaltool
+
+> AudioUnit validation tool for Mac.
+> More information: <https://www.unix.com/man-page/mojave/1/auvaltool>.
```

Comment:
We commonly use https://ss64.com/osx/ for this, but since the manpage isn't mentioned on the site, I think it's ok to leave this URL as-is.

What do you thing @pixelcmtd?

Reply:
Recently, there was a discussion regarding this in #8670 ([Link](https://github.com/tldr-pages/tldr/pull/8670#issuecomment-1264892949)). There isn't a proper manpage (website) with support with all commands for mac. I think the current link is fine so let's see what pixelcmtd thinks.

Reply:
@reinhart1010 https://duckduckgo.com/?q=site%3Ass64.com+auvaltool
We could use https://keith.github.io/xcode-man-pages/auvaltool.1.html instead, but I'd just leave it as is.